### PR TITLE
try to make a bit less deep-review centric

### PR DESCRIPTION
**Pull request by cgreene**
I tried to lightly revise the intro to make it less deep-review centric.

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -2,34 +2,31 @@
...
 
-Therefore, in August 2016 we embarked on the "Deep Review" to survey deep learning in precision medicine.
+In August 2016 we identified the role that deep learning was beginning to play in biomedical research as one such area, and we started an open online effort to survey deep learning's role in precision medicine and to predict its effect in the future.
 In May 2017, the project released a complete review titled "Opportunities and obstacles for deep learning in biology and medicine" [@doi:10.1101/142760].
-Subsequently, we have continued to maintain the project and accept new contributions as the manuscript undergoes traditional peer review at a journal.
-Here we'll discuss our experience leading the Deep Review, including the pros and cons of open collaborative reviews, as well as the infrastructure we created, termed Manubot, to enable open manuscript writing online.
+While the article was under review, we continued to maintain the project and accepted new contributions.
+In January 2018, the article was accepted by the journal that it was initially submitted to.
```
Comment (agitter) [inline content/02.main-text.md]: The provisional acceptance email was February 2018, right?

File: content/02.main-text.md [hunk 2]
```diff
@@ -2,34 +2,31 @@
...
+We'll discuss our experience with the Deep Review project including the pros and cons of open collaborative reviews, as well as the infrastructure we created, termed Manubot, to enable open manuscript writing online.
```
Comment (agitter) [inline content/02.main-text.md]: We haven't introduced the name `Deep Review` yet.  I'm okay introducing the term later, but we should make this more general like `our project` or `our open review`.

File: content/02.main-text.md [hunk 3]
```diff
@@ -2,34 +2,31 @@
...
+We also discuss the extent to which such efforts can contribute to a living and frequently updated literature. `todo: discuss stage 2 maintainers?`
```
Comment (agitter) [inline content/02.main-text.md]: If we want to discuss living literature, then it makes sense to note the stage 2 maintainers.  It's slightly speculative at this point, but we can discuss possibilities for continuing this project.

**Review by agitter — CHANGES_REQUESTED** [review]
Overall, the edits look good to me.  I have two minor suggested changes.

File: content/02.main-text.md (hunk 2 above)
Comment (dhimmel) [reply content/02.main-text.md]: I think we should introduce the name Deep Review, the first time we mention that project above.

File: content/02.main-text.md [hunk 4]
```diff
@@ -2,34 +2,31 @@
...
-## Results
+In total, the project that we termed the "Deep Review" attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
 
 Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
-
 `TODO: need archival issue link`
 In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing the reviews with potential manuscript authorship.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
+To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot) tool for automating manuscript generation.
```
Comment (dhimmel) [inline content/02.main-text.md]: https://github.com/greenelab/manubot-rootstock is currently a better homepage than https://github.com/greenelab/manubut

File: content/02.main-text.md [hunk 5]
```diff
@@ -51,7 +48,7 @@ Conversations during review can reference other pull requests, issues, or author
 Reviewing batches of revisions that focus on a single theme is more efficient than independently discussing isolated comments and edits, and it helps maintain consistent content and tone across different authors and reviewers.
 Once all requested modifications are made, the manuscript maintainers, a subset of authors with elevated GitHub permissions, formally approve the pull request and merge the changes into the official version.
 `TODO: need a figure with a flowchart showing this process`
-The entire process can be orchestrated through GitHub with a web browser if a contributor does not want to use a git client on their own computer.
+The entire process can be orchestrated through GitHub with a web browser.
```
Comment (dhimmel) [inline content/02.main-text.md]: There're some kinds of contributions or PR edits where this is not true. Perhaps we could make it more nuanced to indicate that "most types of contributions" or something along those lines?

Or is this sentence talking only about review and not creating / modifying the repo?

**Review by dhimmel — COMMENTED** [review]
Looking good

File: content/02.main-text.md (hunk 5 above)
Comment (agitter) [reply content/02.main-text.md]: Most of the content editing, including editing or creating new Markdown files, can be managed through the browser.  Is that the scope of the `process` we're discussing here?

Other operations, such as complex merges, certainly require a local git client.

File: content/02.main-text.md (hunk 1 above)
Comment (cgreene) [reply content/02.main-text.md]: Feb 28! 👍  Changed

File: content/02.main-text.md (hunk 3 above)
Comment (cgreene) [reply content/02.main-text.md]: ok - this graduated to a full `TODO`

File: content/02.main-text.md (hunk 5 above)
Comment (cgreene) [reply content/02.main-text.md]: altered the scope lightly

**Review by agitter — APPROVED** [review]

**Review by dhimmel — APPROVED** [review]

**Pull request merged**